src/swanctl/swanctl.opt

```diff
@@ -296,16 +296,18 @@ connections.<conn>.pools =
 	from either the **pools** section or an external pool.
 
 connections.<conn>.if_id_in = 0
-	Default inbound XFRM interface ID for children.
-
-	XFRM interface ID set on inbound policies/SA, can be overridden by child
-	config, see there for details.
+	XFRM interface ID set on inbound policies/SA, can be overridden by child config,
+	see connections.<conn>.children.<child>.if_id_in for valid values,
+	behaviour, and other details.
+	When the ID is set to **%unique** or **%unique-dir** the same if_id is
+	set for the inbound policies for all CHILD_SAs managed by the particular IKE_SA.
```

Comment:
That's just the general behavior of this option, that's already described above, it has nothing to do with `%unique` or `%unique-dir`. Or what's your point here.

Reply:
The point is that the sa,e value is assigned to all CHILD_SAs instead of the value being carried over to the CHILD_SAs verbatim (them all getting %unique or %unique-dir).

Reply:
Ah, I see what you mean. Yes, a unique ID (or two) is currently allocated for each IKE_SA that's based on such a config, which is then used for all CHILD_SAs of a particular IKE_SA (unless the child config overrides it). I've pushed text that describes the behavior to the _1329-number-formats_ branch.

Reply:
Thank you, looks good.